Gulpfile.js

```diff
@@ -195,9 +212,10 @@ function getMatchingFiles(aFileArgs, aFileType)
     return lMatchingFiles;
 }
 
-function getAllTestFiles(aDirectory)
+function getAllTestFiles(aDirectory, aFilter = "test.js")
 {
     const lFiles = [];
+
     getTestsFromDir(aDirectory);
```

Comment:
Remove `getTestsFromDir` and just run the code directly? It's not used anywhere else and `getTestsFromDir` has a reference to `lFiles` so you cannot really pull it out of this func

Reply:
done